base/tps/src/main/java/org/dogtagpki/server/tps/processor/TPSProcessor.java

```diff
@@ -1542,8 +1541,7 @@ protected void fillTokenRecordDefaultPolicy(TokenRecord tokenRecord) throws TPSE
 
         try {
             org.dogtagpki.server.tps.TPSEngine engine = org.dogtagpki.server.tps.TPSEngine.getInstance();
-            TPSSubsystem subsystem = (TPSSubsystem) engine.getSubsystem(TPSSubsystem.ID);
-            IConfigStore configStore = subsystem.getConfigStore();
+            EngineConfig configStore = engine.getConfig();
```

Comment:
The `TPSEngine.getConfig()` does return `TPSEngineConfig`. Any reason for using `EngineConfig` instead of `TPSEngineConfig`?

Reply:
I intend to use the subclass, this branch currently contains a test commit to try figure out why the CI is failing when it passes on master. First test was to use the superclass, as that is the easiest test. Seeing as how that didn't work, I need to do further investigation.